tests/models/test_models_unet_3d_condition.py

```diff
@@ -399,5 +399,23 @@ def test_lora_xformers_on_off(self):
         assert (sample - on_sample).abs().max() < 1e-4
         assert (sample - off_sample).abs().max() < 1e-4
 
+    def test_feed_forward_chunking(self):
+        init_dict, inputs_dict = self.prepare_init_args_and_inputs_for_common()
+        init_dict["norm_num_groups"] = 32
+
+        model = self.model_class(**init_dict)
+        model.to(torch_device)
+        model.eval()
+
+        with torch.no_grad():
+            output = model(**inputs_dict)[0]
+
+        model.enable_forward_chunking()
+        with torch.no_grad():
+            output_2 = model(**inputs_dict)[0]
+
```

Comment:
Maybe add another test for `dim=1` too to be ultra sure, as it's the default for the `TextToVideoSDPipeline` pipeline

Reply:
I would also maybe test for another compatible chunk size to ensure feature robustness.